types/index.d.ts

```diff
@@ -346,6 +347,7 @@ export class Client {
   [custom.http_options]: CustomHttpOptionsProvider;
   [custom.clock_tolerance]: number;
   metadata: ClientMetadata;
+  issuer: Issuer<this>;
```

Comment:
there's also a [`static` version of it](https://github.com/panva/node-openid-client/blob/v3.15.9/lib/client.js#L1565-L1579), please add that too.

Reply:
Done.